snapcraft/cli/_command_group.py

```diff
@@ -61,7 +61,7 @@ def list_commands(self, ctx):
         # Let's keep edit-collaborators hidden until we get the green light
         # from the store.
         commands.pop(commands.index('edit-collaborators'))
-        container_config = env.get_container_config()
-        if not container_config.use_container:
+        build_environment = env.BuilderEnvironmentConfig()
+        if build_environment.provider == 'host':
```

Comment:
Did you mean to use `build_environment.is_host` here?

Reply:
yeah, bummer this retriggers the full suite again :-/